pydantic_ai_slim/pydantic_ai/_agent_graph.py

```diff
@@ -659,11 +671,11 @@ async def process_function_tools(  # noqa: C901
     for call in calls_to_run:
         yield _messages.FunctionToolCallEvent(call)
 
-    user_parts_by_index: dict[int, list[_messages.UserPromptPart]] = defaultdict(list)
```

Comment:
This accidentally reverted some changes in main, can you please verify the PR only includes the new changes?